.github/workflows/ci.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         with:
-          directory: web/cobertura-coverage.xml
+          directory: web/coverage/cobertura-coverage.xml
```

Comment:
@azboss2021 , am not sure if this will work. You could try to use relative path like: `./web/coverage/cobertura-coverage.xml`